UnitsNet.Tests/BaseUnitsTests.cs

```diff
@@ -139,19 +139,24 @@ public void ToStringGivesExpectedResult()
                 TemperatureUnit.Kelvin,
                 AmountOfSubstanceUnit.Mole,
                 LuminousIntensityUnit.Candela);
-
-            UnitAbbreviationsCache cache = UnitsNetSetup.Default.UnitAbbreviations;
-            var m = cache.GetDefaultAbbreviation(LengthUnit.Meter);
-            var kg = cache.GetDefaultAbbreviation(MassUnit.Kilogram);
-            var s = cache.GetDefaultAbbreviation(DurationUnit.Second);
-            var A = cache.GetDefaultAbbreviation(ElectricCurrentUnit.Ampere);
-            var K = cache.GetDefaultAbbreviation(TemperatureUnit.Kelvin);
-            var mol = cache.GetDefaultAbbreviation(AmountOfSubstanceUnit.Mole);
-            var cd = cache.GetDefaultAbbreviation(LuminousIntensityUnit.Candela);
+            
+            var m = LengthUnit.Meter;
+            var kg = MassUnit.Kilogram;
+            var s = DurationUnit.Second;
+            var A = ElectricCurrentUnit.Ampere;
+            var K = TemperatureUnit.Kelvin;
+            var mol = AmountOfSubstanceUnit.Mole;
+            var cd = LuminousIntensityUnit.Candela;
 
             Assert.Equal($"[Length]: {m}, [Mass]: {kg}, [Time]: {s}, [Current]: {A}, [Temperature]: {K}, [Amount]: {mol}, [LuminousIntensity]: {cd}", siBaseUnits.ToString());
```

Comment:
I don't really care much about this, but an alternative format to consider is `L=Meter, M=Kilogram, T=Second, `... etc.